### PR TITLE
Feature/spk 50

### PR DESCRIPTION
**Pull request by AlexMisiulia**

### CODE REVIEW

File: app/src/main/java/com/simplifire/yotinfcclient/utils/CameraHelper.kt [hunk 1]
```diff
@@ -7,14 +7,19 @@ import android.graphics.Bitmap
...
 import java.io.IOException
 import java.text.SimpleDateFormat
 import java.util.Date
 import java.util.Locale
+import javax.inject.Inject
+import javax.inject.Singleton
 
-object CameraHelper {
-    val CAMERA_REQUEST_CODE = 1888
+const val CAMERA_REQUEST_CODE = 1888
```
Comment (cquemin) [inline app/src/main/java/com/simplifire/yotinfcclient/utils/CameraHelper.kt]: This should be split into several parts.
Dealing with an intent to launch the camera should be done in the activity that needs it. We have a camera module that takes picture and uses face detection and everything. For the demo it is not needed but might be worth adding.
Finally the part where the file is saved and transformed should be in an `PhotoInteractor`

File: app/src/main/java/com/simplifire/yotinfcclient/fragment/TakePhotoFragment.kt [hunk 2]
```diff
@@ -59,30 +66,52 @@ class TakePhotoFragment : Fragment() {
     }
 
     private fun initView() {
-        takePhotoActionButton.setOnClickListener {
-            val context = activity ?: return@setOnClickListener
+        continueButton.setOnClickListener {
+            val photoPath = cameraHelper.getPhotoPath()
+            if(photoPath != null)
+            {
```
[the remark this answers is not in the document]
Comment (AlexMisiulia) [reply app/src/main/java/com/simplifire/yotinfcclient/fragment/TakePhotoFragment.kt]: Fixed

File: app/src/main/java/com/simplifire/yotinfcclient/utils/CameraHelper.kt (hunk 1 above)
Comment (AlexMisiulia) [reply app/src/main/java/com/simplifire/yotinfcclient/utils/CameraHelper.kt]: Moved the constant to the Fragment. Agree that I can divide this class into parts, but I am not sure that I should do it now

Comment (cquemin) [reply app/src/main/java/com/simplifire/yotinfcclient/utils/CameraHelper.kt]: I see your point but if you prefer to merge this I won't fight it, but this is the type of things that stayed if not done now.

**Pull request merged**